text/0006-architecture-review-process.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Architecture Review Process
+layout: text
+---
+
+**Version**: 0.1
+
+**Warning**: This document is incomplete. We've added it to the repo before it's finished because we would like a couple of trial runs before we say we're happy, and we would like something to refer to during those reviews.
+
+# Architecture Review Process
+
+## Introduction
+
+Architectural Review decisions within the Firefox organization fall into two categories: "Roadmap" and “Design”.
+
+Product Reviews are not within the scope of this process. This process is about architectural change, which should ultimately be in support of product features, but will generally have an effect on multiple product features. Architectural change can be growing new skills or in reducing large-scale technical debt.
```

Comment:
I'd suggest "skills" → "capabilities", unless you're talking about building people's skills.

Reply:
Fixed in b7d27534